src/main/scala/mesosphere/mesos/RunSpecOfferMatcher.scala

```diff
@@ -26,7 +27,7 @@ object RunSpecOfferMatcher {
     }
 
     val resourceMatchResponse =
-      ResourceMatcher.matchResources(offer, runSpec, otherInstances, ResourceSelector.any(acceptedResourceRoles))
+      ResourceMatcher.matchResources(offer, runSpec, otherInstances, ResourceSelector.any(acceptedResourceRoles), drainingTime)
```

Comment:
Better if `drainingTime` is of `Duratio` time or it name suggest it's in seconds.